Dockerfile.record_thumbnail_attacher

```diff
@@ -14,14 +14,19 @@ RUN npm run build -ws
 
 
 FROM public.ecr.aws/lambda/nodejs:18 as final
+
+ARG AWS_RDS_CERT_BUNDLE
+
 WORKDIR ${LAMBDA_TASK_ROOT}
 
+RUN mkdir /etc/ca-certificates
+RUN echo -e $AWS_RDS_CERT_BUNDLE > /etc/ca-certificates/rds-us-west-2-ca-bundle.pem
+
 COPY --from=builder /usr/local/apps/stela/packages/record_thumbnail_attacher/dist ./packages/record_thumbnail_attacher/dist
 COPY --from=builder /usr/local/apps/stela/packages/record_thumbnail_attacher/package.json ./packages/record_thumbnail_attacher/package.json
 COPY --from=builder /usr/local/apps/stela/packages/logger/dist ./packages/logger/dist
 COPY --from=builder /usr/local/apps/stela/packages/logger/package.json ./packages/logger/package.json
 COPY --from=builder /usr/local/apps/stela/package.json ./package.json
-COPY --from=builder /usr/local/apps/stela/package-lock.json ./package-lock.json
```

Comment:
What was the reason for removing this from the build step?  Depending on the answer it might be worth including in the commit description (or, potentially, having its own commit to explain the reasoning!)

Reply:
I pulled this out when I was trying to figure out what I thought was a dependency versioning issue. That issue turned out to be something else, so I've put this back in.